source/guides/dashboard/introduction.md

```diff
@@ -28,6 +28,13 @@ From the Dashboard you can:
 
 {% imgTag /img/dashboard/dashboard-runs-list.png "Dashboard Screenshot" %}
 
+## Manage runs
+
+From the Dashboard you can:
+
+- Cancel runs currently in progress
```

Comment:
Link to the section describing canceling runs. Use `url` tags described in https://github.com/cypress-io/cypress-documentation/blob/develop/TAGS.md#url
```suggestion
- {% url "Cancel runs" runs#Cancelling-runs %} currently in progress
```